person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
```

Comment:
#- [ **OPTIONAL** ] For best practices I think it would be better if you move the above `GETTERS` and `SETTERS` just before the `initialize()` constructor method.